docs/guides/upgrading-to-indie-linter-v2.md

```diff
@@ -0,0 +1,77 @@
+# Upgrading to Indie Linter v2
+
+This upgrade guide assumes your package uses Indie Linter API v1 and Message API v1.
```

Comment:
These should also probably link to the old documentation (https://github.com/steelbrain/linter/wiki/Linter-Indie-API and https://github.com/steelbrain/linter/wiki/Linter-API#messages).